src/index.html

```diff
@@ -1,13 +1,66 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <div class="product" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="apple imac"
+        class="product__img"
+      >
+      <p class="product__description">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </p>
+      <p class="product__id">Код товара: 195434</p>
+      <div class="product__rating">
+        <div class="product__stars">
+          <img
+            src="images/star-active.svg"
+            alt="star"
+            class="product__stars_star"
+          >
+          <img
+            src="images/star-active.svg"
+            alt="star"
+            class="product__stars_star"
+          >
+          <img
+            src="images/star-active.svg"
+            alt="star"
+            class="product__stars_star"
+          >
+          <img
+            src="images/star-active.svg"
+            alt="star"
+            class="product__stars_star"
+          >
+
+          <img
+            src="images/star.svg"
+            alt="star"
+            lass="product__stars_star"
+          >
+        </div>
+        <a href="#reviews" class="product__reviews">Отзывов: 5</a>
+      </div>
+      <div class="product__price">
+        <p class="product__price_word">Цена:</p>
+        <p class="product__price_result">69 999 грн</p>
+      </div>
+      <!-- <input type="button" value="КУПИТЬ"> -->
+      <button class="product__buy" data-qa="hover">КУПИТЬ</button>
```

Comment:
use text-transform to make this letter in uppercase instead of writing them already in uppercase here